src/wi_stuff.c

```diff
@@ -961,7 +1227,7 @@ static void WI_initNoState(void)
 {
   state = NoState;
   acceleratestage = 0;
-  cnt = 10;
+  cnt = animation ? SHOWNEXTLOCDELAY * TICRATE : 10;
```

Comment:
Is it worth breaking  MBF21 demo compatibility for ID24 intermissions? With `cnt == 10` the "entering" screen is too fast, which is not satisfying.

Reply:
This needs to be resolved across id24 supporting ports. What does @kraflab think?

Reply:
My stance is that we should not accept demos desyncing between mbf21 supporting ports that either support the *strictly cosmetic* features of id24 or not.

Reply:
I'd suggest disabling sync-affecting changes during demo recording / playback.

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1833/commits/1afdb1a0dc6b7ddf3a66b1aefcb762127b633c55 Thanks.